src/components/utils/rhf-inputs/expandable-input/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+export * from './expandable-input';
```

Comment:
You can import it into the index.tsx file of src/component
why you need to create a new file to import this

Reply:
This is the new convention of importing as commons-ui, we will do the same for other components